scripts/build

```diff
@@ -213,6 +213,11 @@ if [ "$PKG_IS_KERNEL_PKG" = "yes" ]; then
   fi
 fi
 
+if [ "$PKG_TOOLCHAIN" = "python" ]; then
+  PKG_DEPENDS_HOST="toolchain distutilscross:host $PKG_DEPENDS_HOST"
+  PKG_DEPENDS_TARGET="toolchain distutilscross:host Python2 $PKG_DEPENDS_TARGET"
```

Comment:
This should still be python2 only

Reply:
Would be nice to have it working with python3 too.

Reply:
What do you mean? build for both Python2 and Python3 at the same time?

Reply:
I mean your package will depend on Python2 only so if I have a python package that is Python3 for the Target it will depend on Python2 which is wrong.